alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/util/SupportedApiSettingsExtensions.kt

```diff
@@ -0,0 +1,33 @@
+package com.amazon.opendistroforelasticsearch.alerting.util
+
+import com.amazon.opendistroforelasticsearch.alerting.core.model.LocalUriInput
+import com.amazon.opendistroforelasticsearch.alerting.elasticapi.convertToMap
+import com.amazon.opendistroforelasticsearch.alerting.settings.SupportedApiSettings
+import com.amazon.opendistroforelasticsearch.alerting.settings.SupportedApiSettings.Companion.validateLocalUriInput
+import org.elasticsearch.action.ActionResponse
+import org.elasticsearch.action.admin.cluster.health.ClusterHealthRequest
+import org.elasticsearch.action.admin.cluster.health.ClusterHealthResponse
+import org.elasticsearch.action.admin.cluster.stats.ClusterStatsRequest
+import org.elasticsearch.action.admin.cluster.stats.ClusterStatsResponse
+import org.elasticsearch.client.Client
+
+fun executeTransportAction(localUriInput: LocalUriInput, client: Client): ActionResponse {
+    val path = validateLocalUriInput(localUriInput)
+    if (path == SupportedApiSettings.CLUSTER_HEALTH_PATH) {
+        return client.admin().cluster().health(ClusterHealthRequest()).get()
+    }
+    if (path == SupportedApiSettings.CLUSTER_STATS_PATH) {
+        return client.admin().cluster().clusterStats(ClusterStatsRequest()).get()
+    }
+    throw IllegalArgumentException("Unsupported API: $path")
```

Comment:
minor:
you can use when statement instead 
```
when (path) {
   is SupportedApiSettings.CLUSTER_HEALTH_PATH -> return ..
   is SupportedApiSettings.CLUSTER_STATS_PATH -> return ...
   else -> throw IllegalArgumentException
}
```

Reply:
Thank you for the tip! The syntax is a little different from your example due to some compiler warnings, but I refactored this to `when`.